### PR TITLE
Clean up unnecessary CodeGenerator phase and code 

### PR DESCRIPTION
**Pull request by rudy2steiner**
Remove LateSequentialConstantStoreSimplification phse and unnecessary code invoke

Fixes: #10181
Fixes: #10180

Signed-off-by: rudy2steiner rudy_steiner@163.com

### CODE REVIEW

**Comment by 0xdaryl** [comment]
Thanks for the contribution!

* Please be sure you've signed the ECA per the project's legal requirements -> https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md#legal-considerations

* Please be sure the forward copyright date in the file header comment is updated to 2020 in all files you've changed

* As you are addressing two separate issues in this PR, please split the changes for each into their own commit (you can still use one PR for both issues, but different commits).

**Comment by rudy2steiner** [comment]
I met a exception when sign eca
```
Error, An error has occured and your Eclipse Contributor Agreement was not submitted. (ECA-04)
```

**Comment by 0xdaryl** [comment]
There have been a very small number of reports in the past about problems signing the ECA.  To my knowledge all of these went away with the short passage of time.  Please try again later.

Unfortunately, signing the ECA is an Eclipse Foundation process (not controlled or managed by OpenJ9) so if the problems persist for you the next course of action is to inquire with the Eclipse Foundation for assistance (e.g., through the Eclipse Community Forum).

**Comment by rudy2steiner** [comment]
Ok, i will try to contact with eclipse email or Forum

**Comment by rudy2steiner** [comment]
@0xdaryl would you mind help me review this work!

File: runtime/compiler/codegen/J9CodeGenPhaseFunctionTable.hpp [hunk 1]
```diff
@@ -34,5 +34,4 @@
    TR::CodeGenPhase::performMoveUpArrayLengthStoresPhase,                                    //MoveUpArrayLengthStoresPhase
    TR::CodeGenPhase::performInsertEpilogueYieldPointsPhase,                                  //InsertEpilogueYieldPointsPhase
    TR::CodeGenPhase::performCompressedReferenceRematerializationPhase,
-   TR::CodeGenPhase::performIdentifyUnneededByteConvsPhase,                                  //IdentifyUnneededByteConvsPhase
-   TR::CodeGenPhase::performLateSequentialConstantStoreSimplificationPhase,                  //LateSequentialConstantStoreSimplificationPhase
+   TR::CodeGenPhase::performIdentifyUnneededByteConvsPhase,                                  //IdentifyUnneededByteConvsPhase
```
Comment (0xdaryl) [inline runtime/compiler/codegen/J9CodeGenPhaseFunctionTable.hpp:37]: Missing EOL here.

**Review by 0xdaryl — CHANGES_REQUESTED** [review]
My apologies--this PR seems to have slipped through the cracks.  Code changes generally look good.  A couple of minor points:

* Please shorten the PR title so that it is not truncated
* Please capitalize the first word in each commit title per the Contributor Guidelines
* Update all forward copyrights to 2021

**Comment by rudy2steiner** [comment]
@0xdaryl  Thanks for you feedbacks, updated!

**Comment by 0xdaryl** [comment]
Jenkins test sanity all jdk8,jdk11

**Comment by 0xdaryl** [comment]
Jenkins test sanity osx jdk8

**Review by 0xdaryl — APPROVED** [review]

**Comment by 0xdaryl** [comment]
Jenkins test sanity osx jdk8

**Pull request merged**